Update 08_mtk_command_options.mdx (#4063)
EnterpriseDB/docs: product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

`-filterProp <file_name>`

 `<file_name>` specifies the name of a file that contains constraints in key=value pairs. Each record read from the database is evaluated against the constraints. Those that satisfy the constraints are migrated. 
 
 The left side of the pair lists a table name: 
 - The table name must not be schema qualified. 

>  - Oracle table names are not case sensitive, but Oracle defaults to uppercase. Postgres, on the other hand defaults to lowercase. When migrating Oracle data, make sure that the case you are using matches the case in Oracle; otherwise, Migration Toolkit can't match it and ignores the constraint. 

Reviewer: ```suggestion
 - Oracle table names are not case sensitive, but Oracle defaults to uppercase. Postgres, on the other hand, defaults to lowercase. When migrating Oracle data, make sure that the case you are using matches the case in Oracle; otherwise, Migration Toolkit can't match it and ignores the constraint. 
```

>  - Oracle table names are not case sensitive, but Oracle defaults to uppercase. Postgres, on the other hand, defaults to lowercase. When migrating Oracle data, make sure that the case you are using matches the case in Oracle; otherwise, Migration Toolkit can't match it and ignores the constraint. 

Reviewer: ```suggestion
 - Oracle table names are not case sensitive, but Oracle defaults to uppercase. Postgres, on the other hand, defaults to lowercase. When migrating Oracle data, ensure the case you are using matches the case in Oracle; otherwise, Migration Toolkit can't match it and ignores the constraint. 
```

 
The right side specifies a condition that must be true for each row migrated.  

For example, 

`COUNTRIES=COUNTRY_ID<>'AR'`

> migrates only those countries with a `country_id` value that is not equal to `AR`. This constraint applies to the countries table. 

Reviewer: ```suggestion
migrates only those countries with a `COUNTRY_ID` value that is not equal to `AR`. This constraint applies to the countries table. 
```

> migrates only those countries with a `COUNTRY_ID` value that is not equal to `AR`. This constraint applies to the countries table. 

Reviewer: ```suggestion
migrates only those countries with a `COUNTRY_ID` value that is not equal to `AR`. This constraint applies to the COUNTRIES table. 
```
